### ISSUE TITLE
Indent with spaces

### ISSUE THREAD

**Issue by mkroening** [hermitcore/libhermit-rs#417]
Reformatting commits can be hidden from blame: [Ignore commits in the blame view
](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view)

**Comment by TaeminHa** [comment]
Hi! I'm a student from UT Austin, and would be happy to take on this issue! Can you assign this to me?

**Comment by mkroening** [comment]
I don't think this is a good first issue. This is mainly a reminder to discuss this move internally or drop it. Doing the conversion itself will be fairly straightforward.
